common/src/main/java/tw/edu/ntut/sce/java18/common/utils/CreateRandomRoomState.java

```diff
@@ -0,0 +1,38 @@
+package tw.edu.ntut.sce.java18.common.utils;
+
+import java.util.ArrayList;
+
+public class CreateRandomRoomState {
+
+  public ArrayList<String> randomRoomData() {
```

Comment:
除非有特別理由，不然建議使用父類別當作回傳值型態
```suggestion
  public List<String> randomRoomData() {
```

Reply:
這個code是我不小心commit上去的,
目前已經不用l ist ,也改成void了,
看之後改動後的內容走向,也可能會重新命名為 serviceimpl 

Ing